### PR TITLE
FE: fix alignment of icons and add feature flag for settings button

### PR DESCRIPTION
**Pull request by smonero**
### Description

Since settings button isn't working yet we can gate it using a feature flag. We also use flex to put the icons at the same height.
before:
Screen Shot 2022-04-29 at 10 55 19 PM

after:

Screen Shot 2022-04-29 at 10 53 16 PM

edit: might not need the feature flag because https://github.com/lyft/clutch/pull/2219 will probably be merged in soon

Also fixed the text to be `noWrap` to prevent wrapping of long links, and set spacing to 1 for that grid in the corner.

### GitHub Issue

Fixes #

### CODE REVIEW

File: frontend/workflows/projectCatalog/src/details/index.tsx [hunk 1]
```diff
@@ -39,11 +39,13 @@ const QuickLinksAndSettingsBtn = ({ linkGroups }) => {
         <Grid item>
           <QuickLinksCard linkGroups={linkGroups} />
         </Grid>
-        <Grid item style={{ padding: "10px" }}>
-          <IconButton onClick={() => {}}>
-            <SettingsIcon />
-          </IconButton>
-        </Grid>
+        <SimpleFeatureFlag feature="projectSettings">
```
Comment (jdslaugh) [inline frontend/workflows/projectCatalog/src/details/index.tsx:42]: Can we rename this to projectCatalogSettings? That way its inline with the projectCatalog feature flag

**Review by jdslaugh — APPROVED** [review]

File: frontend/workflows/projectCatalog/src/details/index.tsx [hunk 2]
```diff
@@ -35,15 +35,26 @@ const DisabledItem = ({ name }: { name: string }) => (
...
     <>
-      <Grid container direction="row" style={{ padding: "8px", justifyContent: "flex-end" }}>
+      <Grid
+        container
+        direction="row"
+        style={{
+          padding: "8px",
+          justifyContent: "flex-end",
+          display: "flex",
+          alignItems: "center",
```
Comment (dschaller) [inline frontend/workflows/projectCatalog/src/details/index.tsx:45]: Grid should provide props for justify and align and flex should be set automatically.

Comment (smonero) [reply frontend/workflows/projectCatalog/src/details/index.tsx:45]: when I remove them I get this issue (even with adding justify="flex-end" for the grid component

Comment (smonero) [reply frontend/workflows/projectCatalog/src/details/index.tsx:53]: oh nvm I get it. I'll fix it now

File: frontend/workflows/projectCatalog/src/details/quick-links.tsx [hunk 3]
```diff
@@ -94,15 +94,17 @@ const QuickLinkGroup = ({ linkGroupName, linkGroupImage, links }: QuickLinkGroup
...
       >
         {validLinks.map(link => (
           <PopperItem key={link.name}>
             {link?.url && (
               <Link href={link.url}>
-                <Typography color="inherit" variant="body4">
-                  {link.name}
-                </Typography>
+                <span style={{ whiteSpace: "nowrap" }}>
+                  <Typography color="inherit" variant="body4">
```
Comment (dschaller) [inline frontend/workflows/projectCatalog/src/details/quick-links.tsx:104]: Do we want to remove the color from Link here?

**Review by dschaller — APPROVED** [review]

**Pull request merged**